navigation/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/turnlane/TurnLaneAdapter.java

```diff
@@ -55,22 +54,24 @@ public void addTurnLanes(IntersectionLanes[] lanes, String maneuverModifier) {
     notifyDataSetChanged();
   }
 
-  private void setLaneImage(TurnLaneViewHolder holder, IntersectionLanes lane) {
+  private void setLaneImage(TurnLaneViewHolder holder, IntersectionLanes lanes) {
     StringBuilder builder = new StringBuilder();
 
     // Indications
-    if (lane.getIndications() != null) {
-      for (String indication : lane.getIndications()) {
+    if (lanes.getIndications() != null) {
+      for (String indication : lanes.getIndications()) {
         builder.append(indication);
       }
-
-      if (builder.toString().contains(TURN_LANE_INDICATION_STRAIGHT)
-        || (builder.toString().contains(TURN_LANE_INDICATION_NONE) && lane.getValid())) {
+      if (builder.toString().contains(TURN_LANE_INDICATION_STRAIGHT) && lanes.getValid()) {
```

Comment:
I'm not sure I understand what this check for `TURN_LANE_INDICATION_STRAIGHT` does.

Reply:
If the lane contains indication `"straight"` and is valid, then we append the modifier, which will allow us to retrieve the asset from the map with multiple arrows and (one dark, one greyed out).  This the issue stemming from not having completely greyed out assets